packaging/opensuse/snapd.spec

```diff
@@ -171,6 +167,28 @@ pushd %{indigo_srcdir}
 # Add patch0 -p1 ... as appropriate here.
 popd
 
+# Generate snapd.defines.mk, this file is included by snapd.mk. It contains a
+# number of variable definitions that are set based on their RPM equivalents.
+# Since we can apply any conditional overrides here in the spec file we can
+# maintain one consistent set of variables across the spec and makefile worlds.
+cat >snapd.defines.mk <<__DEFINES__
+# This file is generated by openSUSE's snapd.spec
+# Directory variables.
+prefix = %{_prefix}
+bindir = %{_bindir}
+sbindir = %{_sbindir}
+libexecdir = %{_libexecdir}
+mandir = %{_mandir}
+datadir = %{_datadir}
+localstatedir = %{_localstatedir}
+unitdir = %{_unitdir}
+# Build configuration
+with_core_bits = 0
+with_alt_snap_mount_dir = %{!?with_alt_snap_mount_dir:0}%{?with_alt_snap_mount_dir:1}
```

Comment:
Would it make sense to simply pass %snap_mount_dir here?

Reply:
I decided against that because that gives the feeling of this being any directory we want. That is not the case and we want to really only support one of two locations.

Reply:
I see, sure. Thanks